tests/test_config.py

```diff
@@ -25,38 +25,32 @@ class TestConfig(TestCase):
     def setUp(self):
         """Set up. Creates a fake config"""
         self.config_test = _Config()
+        if os.path.exists(config.FILEPATH):
```

Comment:
Cleanup should not be done in the setUp, but in the tearDown. You need to END the tests with everything clean.